packages/wallet/frontend/src/lib/api/account.ts

```diff
@@ -13,7 +13,7 @@ export const fundAccountSchema = z.object({
     .number({
       invalid_type_error: 'Please enter a valid amount'
     })
-    .min(1, { message: 'Please enter an amount' })
+    .min(0, { message: 'Please enter an amount' })
     .positive()
```

Comment:
[Zod numbers](https://zod.dev/?id=numbers).
```suggestion
    .nonnegative({ message: 'Please enter an amount' })
```